src/styles/product-card.css

```diff
@@ -0,0 +1,127 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 50px;
+}
+
+.product-card__image {
+  content: url(../images/imac.jpeg);
+  width: 160px;
+  height: 134px;
+  margin: 30px 0;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  width: 170px;
+  margin: 15px 0 0;
+}
+
+.product-card__code {
+  font-style: normal;
+  font-weight: normal;
+  width: 170px;
+  font-size: 11px;
+  line-height: 13px;
+  margin: 5px 0;
+}
+
+.product-card__rating {
+  display: flex;
+  width: 170px;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: top;
+  margin-top: 10px;
+}
+
+.product-card__stars * {
+  display: inline;
+  padding: 1px;
+}
+
+.product-card__stars > * {
+  content: url(../images/star.svg);
+}
+
+.product-card__stars--5 :nth-child(-n+5),
+.product-card__stars--4 :nth-child(-n+4),
+.product-card__stars--3 :nth-child(-n+3),
+.product-card__stars--2 :nth-child(-n+2),
+.product-card__stars--1 :first-child {
+  content: url(../images/star-active.svg);
+}
+
+.product-card__review {
+  display: inline-flex;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 15px;
+  text-align: right;
+  margin: 0;
```

Comment:
this rule appears to many times, add for all p elements

Reply:
+